inputfiles/addedTypes.json

```diff
@@ -396,45 +396,6 @@
                     }
                 }
             },
-            "ImageBitmapOptions": {
```

Comment:
What replaces this entry? Whatever it is, it results in a whole bunch of unwanted type aliases.

Reply:
The [already existing `dictionary ImageBitmapOptions`](https://github.com/Microsoft/TSJS-lib-generator/blob/5239b2477c826954e2c2e515a2a19432116fc309/inputfiles/idl/HTML%20-%20ImageBitmap%20and%20animations.widl#L17) in `HTML - ImageBitmap and animations.widl` replaces this entry. This was a conflict if I remember right.